docs/cni-plugin.md

```diff
@@ -35,16 +35,29 @@ Another example with a trunk port and jumbo frames:
 }
 ```
 
+Another example with QinQ configured:
+
+```json
+{
+    "name": "myqinqnet",
+    "type": "ovs",
+    "bridge": "mynet2",
+    "vlan": 1000,
+    "cvlan": [ { "id" : 24 }, { "minID" : 100, "maxID" : 102 } ]
+}
+```
+
 ## Network Configuration Reference
 
 * `name` (string, required): the name of the network.
 * `type` (string, required): "ovs".
 * `bridge` (string, required): name of the bridge to use.
 * `vlan` (integer, optional): VLAN ID of attached port. Trunk port if not
-   specified.
+  specified. If CVLAN IDs are provided this VLAN ID represents SVLAN.
 * `mtu` (integer, optional): MTU.
 * `trunk` (optional): List of VLAN ID's and/or ranges of accepted VLAN
   ID's.
+* `cvlan` (optional): List of customer used VLAN ID's.
```

Comment:
You can drop the `used`.
I suggest you also explain that having this key implies that the port is using the vlan dot1q-tunnel mode.

Reply:
Updated, would you take another look to see if it's better?